### ISSUE TITLE
Convert imgur page URLs to imgur image URLs

### ISSUE THREAD

**Issue by sparkertime** [citizenparker/swsheets#42]
When adding an image to a character, several folks have accidentally used the imgur page URL (which loads an HTML page) instead of the imgur image URL (which actually loads the image). This doesn't work as the page isn't an image.

As much as I would like to ask them to use the right URL, enough people are doing this that I should just do the conversion on their behalf when they do the wrong thing.

**Comment by bknowles** [comment]
:+1:  :100:

**Comment by sparkertime** [comment]
I should have been more specific with this :( The error folks have committed have been posting `http://imgur.com/0UphWm5` instead of `http://i.imgur.com/a4JGel.png`

I would update this to apply to any imgur URL that doesn't start with `i.imgur...`

**Comment by chrisjpowers** [comment]
Ah, sorry, didn't see your note from before. One additional problem I realize is that currently `.jpg` is hardcoded, but obviously that won't work for pngs and gifs.

It looks like imgur includes this tag in its HTML:

``` html
<link rel="image_src" href="http://i.imgur.com/0UphWm5.png"/>
```

The best solution would possibly be to fetch the HTML payload of the page they type in, parse it, and grab this `href` value.

**Comment by chrisjpowers** [comment]
Okay, I implemented this in #61
